helm-chart/splunk-enterprise/templates/extra-manifests.yaml

```diff
@@ -0,0 +1,4 @@
+{{- range .Values.extraManifests }}
```

Comment:
@joshuasimon-taulia Has this been tested? Any proof of the same?

Reply:
using default values
```
➜  splunk-operator git:(extra-manifests) helm template $(basename $PWD) $PWD  --validate
➜  splunk-operator git:(extra-manifests) echo $?
0
```

values-test.yaml
```
extraManifests:
 - apiVersion: cloud.google.com/v1beta1
   kind: BackendConfig
   metadata:
     name: "{{ .Release.Name }}-test"
   spec:
     securityPolicy:
       name: "gcp-cloud-armor-policy-test"
```

```
➜  splunk-operator git:(extra-manifests) helm template $(basename $PWD) $PWD  --validate -f values-test.yaml
---
# Source: splunk-operator/templates/extra-manifests.yaml
apiVersion: cloud.google.com/v1beta1
kind: BackendConfig
metadata:
  name: 'splunk-operator-test'
spec:
  securityPolicy:
    name: gcp-cloud-armor-policy-test
```

Reply:
is there a helm testing pattern for this repo?

Reply:
I have added kuttl test cases in another PR

Reply:
I have also tested these changes and they work fine